base/reflection.jl

```diff
@@ -169,7 +169,7 @@ julia> fieldnames(Rational)
 (:num, :den)
 ```
 """
-fieldnames(t::DataType) = (fieldcount(t); # error check to make sure type is specific enough
+@pure fieldnames(t::DataType) = (fieldcount(t); # error check to make sure type is specific enough
                            (_fieldnames(t)...,))
```

Comment:
The whitespace here should be increased by the same number of characters added on the line above.

Reply:
I think when we go multiline, we prefer to use the longer `function `syntax anyway.

Reply:
Ok